skin_meta_api.lua

```diff
@@ -60,6 +60,18 @@ function skin_class:get_preview()
 end
 
 function skin_class:apply_skin_to_player(player)
+
+	local function concat_texture(base, ext)
+		if base == "blank.png" then
+			return ext
+		elseif ext == "blank.png" then
+			return base
+		else
```

Comment:
`else` is redundant when all cases above return.

Reply:
else is for the case the base is not blank.png AND ext is not blank.png

Reply:
In that case `return base .. "^" .. ext` would be executed anyway, because that does not have any condition.

Reply:
Ah, ok ;-) 
I like the "if" / "elseif" chains syntax as is, as replacement for not supported "case" command in lua. I think there is no difference in performance or execustion, just developers coding-stlye taste. So I keep the code as is